trace/opencensus/trace/reporters/zipkin_reporter.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Export the spans data to Zipkin Collector."""
+
+import datetime
+import json
+import logging
+import requests
+
+DEFAULT_ENDPOINT = '/api/v1/spans'
```

Comment:
I think you are using the new format, which would be /api/v2/spans https://github.com/openzipkin/zipkin-api/pull/33